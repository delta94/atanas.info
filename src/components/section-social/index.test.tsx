import * as React from 'react';
import { shallow } from 'enzyme';

import { SectionSocial } from '.';

describe('SectionSocial component', () => {
	it('Should render the SectionSocial component', () => {
		const wrapper = shallow(<SectionSocial />);

		expect(wrapper).toMatchSnapshot();
	});
});
