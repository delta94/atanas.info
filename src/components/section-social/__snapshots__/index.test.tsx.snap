// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionSocial component Should render the SectionSocial component 1`] = `
<Component
  hasButton={true}
  hasShell={false}
  id="social"
>
  <header
    className="c-section__head"
  >
    <div
      className="o-shell"
    >
      <h1>
        Social
      </h1>
    </div>
  </header>
  <div
    className="c-section__entry"
  >
    <div
      className="o-shell"
    >
      <div
        className="o-grid"
      >
        <div
          className="o-grid__item o-grid__item--1of2"
        >
          <h2>
            Codersrank Profile
          </h2>
          <codersrank-widget
            username="scriptex"
          />
        </div>
        <div
          className="o-grid__item o-grid__item--1of2"
        >
          <h2>
            LinkedIn Profile
          </h2>
          <div
            className="LI-profile-badge"
            data-locale="en_US"
            data-size="large"
            data-theme="light"
            data-type="horizontal"
            data-vanity="scriptex"
            data-version="v1"
          >
            <Component
              className="LI-simple-link"
              href="https://bg.linkedin.com/in/scriptex?trk=profile-badge"
            >
              Atanas Atanasov
            </Component>
          </div>
        </div>
        <div
          className="o-grid__item o-grid__item--1of2"
        >
          <h2>
            Twitter Profile
          </h2>
          <div
            className="twitter-timeline-wrapper"
          >
            <a
              className="twitter-timeline"
              href="//twitter.com/scriptexbg?ref_src=twsrc%5Etfw"
            />
          </div>
        </div>
        <div
          className="o-grid__item o-grid__item--1of2"
        >
          <h2>
            YouTube Channel
          </h2>
          <iframe
            className="youtube-wrapper"
            height="400"
            loading="lazy"
            src="//www.youtube.com/embed/?listType=user_uploads&list=scriptex"
          />
        </div>
      </div>
    </div>
  </div>
</Component>
`;
